network-config.sample

```diff
@@ -0,0 +1,11 @@
+development:
```

Comment:
ocean.py now has a test on polygon network as well. [Here](https://github.com/oceanprotocol/ocean.py/blob/main/tests/integration/remote/test_polygon.py). 
- Do we  account for that? 
- For starters, perhaps rename this file to include "mumbai". 
- And if appropriate, have another file for polygon.

Also, is ".sample" in the filename a standard extension? Seems weird.

Reply:
I added the polygon entry. Yes, I used an older convention I remember, from env var files and other such files which you can edit and replace, but they need to be copied or activated somehow. I added this to the CI flow too, and we should add it to the documentation (documenting that in the phase 2 issue)